tools/driver/p4c_src/main.py

```diff
@@ -166,6 +171,10 @@ def main():
     user_defined_version = os.environ.get('P4C_DEFAULT_VERSION')
     if user_defined_version != None:
         opts.language = user_defined_version
+    # accept multiple ways of specifying which language, and ensure that it is a consistent
```

Comment:
I think this code would be even more useful in the C++ compiler.

Reply:
I can add it to the backend as well, however, for anything a bit more complicated, running the driver is much more friendly than directly running the backend. And the driver normalizes the params, so the backend arguments are insulated from changes.